Classes/Form/Field/Text.php

```diff
@@ -36,6 +36,16 @@ class Text extends Input implements FieldInterface {
 	protected $enableRichText = FALSE;
 
 	/**
+	 * @var string
+	 */
+	protected $renderType;
```

Comment:
Is there a reasonable default value we can apply here, e.g. the $renderType that results in a standard `<textarea>`?